backend/test/mocks/MafiaGameMock.js

```diff
@@ -45,29 +55,54 @@ module.exports.createMafiaGameWithOnePlayerMock = function (port) {
     });
 
     return { io, mafiaGame, socketIOServer: server, roomID, hostPlayer };
-};
+}
 
-module.exports.addPlayer = function (player, roomID) {
+function addPlayer(player, roomID) {
     let room = mafiaGame.gameRoomsDict[roomID];
 
     room.addPlayer(player);
-};
+}
 
-module.exports.addMafiaVote = function (voter, votedFor, roomID) {
+function addMafiaVote(voter, votedFor, roomID) {
     const room = mafiaGame.gameRoomsDict[roomID];
     const { mafiaVoteMap } = room.voteHandler;
 
-    mafiaVoteMap[voter] = votedFor;
-};
+    mafiaVoteMap[voter.nickname] = votedFor;
+}
 
-module.exports.addPlayers = function (players, roomID) {
+function addTrialVote(voter, votedFor, roomID) {
+    const room = mafiaGame.gameRoomsDict[roomID];
+    const { trialVoteMap } = room.voteHandler;
+
+    trialVoteMap[voter.nickname] = votedFor;
+}
+
+function addPlayers(players, roomID) {
     const room = mafiaGame.gameRoomsDict[roomID];
     for (let player of players) {
         room.addPlayer(player);
     }
-};
+}
 
-module.exports.switchPlayer = function (nickname, roomID) {
+function switchHostPlayer(nickname, roomID) {
     const room = mafiaGame.gameRoomsDict[roomID];
-    testPlayerSocket.player = room.getPlayerByNickname(nickname);
+    testHostPlayerSocket.player = room.getPlayerByNickname(nickname);
+}
+
+function resetRoom(roomID) {
+    const room = mafiaGame.gameRoomsDict[roomID];
+    room.resetGame();
+    room.players = [];
+    room.host = null;
+    room.voteHandler.resetVotes();
+}
+
+module.exports = {
```

Comment:
Changed the way we exported the functions in the mock class, to be consistent with how we've done it elsewhere.